source/jsonnet/england-wales/lib/common_rules.libsonnet

```diff
@@ -1,84 +1,100 @@
-{
-  over19: {
-    id: 'date-of-birth-answer',
-    condition: 'less than or equal to',
-    date_comparison: {
-      value: 'now',
-      offset_by: {
-        years: -19,
-      },
+local over19 = {
+  id: 'date-of-birth-answer',
+  condition: 'less than or equal to',
+  date_comparison: {
+    value: std.extVar('census_date'),
+    offset_by: {
+      years: -19,
     },
   },
-  over16: {
-    id: 'date-of-birth-answer',
-    condition: 'less than or equal to',
-    date_comparison: {
-      value: 'now',
-      offset_by: {
-        years: -16,
-      },
+};
+
+local over16 = {
+  id: 'date-of-birth-answer',
+  condition: 'less than or equal to',
+  date_comparison: {
+    value: std.extVar('census_date'),
+    offset_by: {
+      years: -16,
     },
   },
-  under16: {
-    id: 'date-of-birth-answer',
-    condition: 'greater than',
-    date_comparison: {
-      value: 'now',
-      offset_by: {
-        years: -16,
-      },
+};
+
+local under16 = {
+  id: 'date-of-birth-answer',
+  condition: 'greater than',
+  date_comparison: {
+    value: std.extVar('census_date'),
+    offset_by: {
+      years: -16,
     },
   },
-  over15: {
-    id: 'date-of-birth-answer',
-    condition: 'less than or equal to',
-    date_comparison: {
-      value: 'now',
-      offset_by: {
-        years: -15,
-      },
+};
+
+local over15 = {
+  id: 'date-of-birth-answer',
+  condition: 'less than or equal to',
+  date_comparison: {
+    value: std.extVar('census_date'),
+    offset_by: {
+      years: -15,
     },
   },
-  under5: {
-    id: 'date-of-birth-answer',
-    condition: 'greater than',
-    date_comparison: {
-      value: 'now',
-      offset_by: {
-        years: -5,
-      },
+};
+
+local under5 = {
+  id: 'date-of-birth-answer',
+  condition: 'greater than',
+  date_comparison: {
+    value: std.extVar('census_date'),
+    offset_by: {
+      years: -5,
     },
   },
-  under4: {
-    id: 'date-of-birth-answer',
-    condition: 'greater than',
-    date_comparison: {
-      value: 'now',
-      offset_by: {
-        years: -4,
-      },
+};
+
+local under4 = {
+  id: 'date-of-birth-answer',
+  condition: 'greater than',
+  date_comparison: {
+    value: std.extVar('census_date'),
+    offset_by: {
+      years: -4,
     },
   },
-  under3: {
-    id: 'date-of-birth-answer',
-    condition: 'greater than',
-    date_comparison: {
-      value: 'now',
-      offset_by: {
-        years: -3,
-      },
+};
+
+local under3 = {
+  id: 'date-of-birth-answer',
+  condition: 'greater than',
+  date_comparison: {
+    value: std.extVar('census_date'),
+    offset_by: {
+      years: -3,
     },
   },
-  under1: {
-    id: 'date-of-birth-answer',
-    condition: 'greater than',
-    date_comparison: {
-      value: 'now',
-      offset_by: {
-        years: -1,
-      },
+};
+
+local under1 = {
```

Comment:
These can all go back to inline definitions (no `local`) now that census date is not being passed in.